modules/nodes-nodes-kernel-arguments.adoc

```diff
@@ -162,11 +162,16 @@ Starting pod/ip-10-0-141-105ec2internal-debug ...
 To use host binaries, run `chroot /host`
 
 sh-4.2# cat /host/proc/cmdline
-BOOT_IMAGE=/ostree/rhcos-... console=tty0 console=ttyS0,115200n8
+BOOT_IMAGE=/ostree/rhcos-... ,115200n8
```

Comment:
`,115200n8` is part of the `console=` argument and should be removed too.

Reply:
Whoops, https://github.com/openshift/openshift-docs/pull/29525#discussion_r578793753 is correct.  Disregard.